### DISCUSSION TITLE
displacement

### DISCUSSION THREAD

**Discussion by brucebra** [arkypita/LaserGRBL#1864] [General]
hello all,
I'm having trouble with the outline moving in the x-direction from the hatch about every 4th time of lasering.

I edit my drawing in Inksape and delete everything there except the hatching. I save this as an SVG and open it in LaserGRBL. Then, in the preview I see that an outline has "emerged" around the hatching.

I would be interested to know how I can prevent this line from appearing. Maybe there are mistakes in my settings in LaserGRBL:

 -----------------------------------
$0=10 (Step pulse time)
$1=25 (Step idle delay)
$2=0 (Step pulse invert)
$3=0 (Step direction invert)
$4=0 (Invert step enable pin)
$5=0 (Invert limit pins)
$6=0 (Invert probe pin)
$10=3 (Status report options)
$11=0.010 (Junction deviation)
$12=0.002 (Arc tolerance)
$13=0 (Report in inches)
$20=0 (Soft limits enable)
$21=0 (Hard limits enable)
$22=0 (Homing cycle enable)
$23=0 (Homing direction invert)
$24=25.000 (Homing locate feed rate)
$25=500.000 (Homing search seek rate)
$26=250 (Homing switch debounce delay)
$27=1.000 (Homing switch pull-off distance)
$100=80.000 (X-axis travel resolution)
$101=80.000 (Y-axis travel resolution)
$102=400.000 (Z-axis travel resolution)
$110=5000.000 (X-axis maximum rate)
$111=5000.000 (Y-axis maximum rate)
$112=5000.000 (Z-axis maximum rate)
$120=200.000 (X-axis acceleration)
$121=200.000 (Y-axis acceleration)
$122=200.000 (Z-axis acceleration)
$130=5000.000 (X-axis maximum travel)
$131=5000.000 (Y-axis maximum travel)
$132=5000.000 (Z-axis maximum travel)

----------------------------
the picture is showing the offset mistake.

Hopefully You have tipps for me.

Laser: Eleksmaker A3, 2.500 mW blue Laser

KR
Andreas

![displacement](https://user-images.githubusercontent.com/32128223/178974919-f036a42f-e492-48ed-8cb1-a38020fa1d61.jpg)

**Comment by Tribo-2dot0** [comment]
First of all you should update your grbl to 1.1, it looks like you are using an old version with no laser support. The displacement check the tightness of the belts and eccentric nuts.
